tests/fixtures/points.py

```diff
@@ -11,43 +11,97 @@
 from tests.fixtures.payload import one_random_payload_please
 
 
+import numpy as np
+from numpy.linalg import norm
+import warnings
+
+def too_close(vectors: np.ndarray, distance: str, thold : float= 10e-5) -> bool:
+    """
+    Efficiently checks if any two distances between the last vector and the others
+    differ by less than or equal to `thold`.
+
+    Parameters:
+        vectors (np.ndarray): Array of shape (N, D)
+        distance (str): One of {'text', 'image', 'code'}
+        thold (float): Threshold for minimal distance difference
+
+    Returns:
+        bool: True if any two distances are within threshold, else False
+    """
```

Comment:
relying on vector names seems to be off
is it possible to rely on distance type?